service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -96,23 +104,29 @@ public ProcessorImpl(DSLContext db) {
     }
   }
 
+  // Applicants
+
   public List<ApplicantReturn> getAllApplicants() {
-    List<Applicants> applicants = db.selectFrom(Tables.APPLICANTS).fetchInto(Applicants.class);
-    return applicants.stream()
-        .map(applicant -> new ApplicantReturn(applicant.getUserId(), applicant.getResume(), applicant.getFileType(),
-            Arrays.copyOf(applicant.getInterests(), applicant.getInterests().length, String[].class),
-            applicant.getPriorInvolvement(), applicant.getWhyJoin()))
-        .collect(Collectors.toList());
+    List<ApplicantReturn> applicants = db.selectFrom(Tables.APPLICANTS).fetchInto(ApplicantReturn.class);
```

Comment:
Make sure you test this, I usually only call into for a jOOQ generated class but if this works then great